appgate/openapi/parser.py

```diff
@@ -566,7 +553,7 @@ def make_attrib_maker(self, attrib_maker_config: AttribMakerConfig) -> AttribMak
                     source_field=attrib_maker_config.definition["x-size-source"],
                 )
             elif attrib_name == "id":
-                return IdAttribMaker(
+                return AttribMaker(
                     name=attrib_name,
                     tpe=TYPES_MAP[tpe],
                     base_tpe=TYPES_MAP[tpe],
```

Comment:
I think we can let this fall through to the `else` block (on line 611) without the ```default=DEFAULT_MAP.get(tpe) if format != "uuid" else None```

Reply:
:+1: